File: pset1/mario.c
// Program that prints out a pyramid from the Mario game of a height specified by the user

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; 

    do 
    {
        // Get input
        height = get_int("Height: ");
    } 
    // check correctness input
    while (height < 1 || height > 8);
    
    // Run loop until height (inclusive) is reached
    for (int i = 1; i <= height; i++)
    {        
        
        for (int j = 0; j < height; j++)       
        {   
            // Print left side of pyramid
            if (j < (height - i)) 
            {
                printf(" ");
            }
            else 
            {
                printf("#");
            }
        }

        // Print gap
        printf("  ");

        // Print right side of pyramid
        for (int j = 0; j < i; j++)        
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/mario/more/mario.py
# Get height
while True:
    height = input("Height: ")
    try:
        height = int(height)
    except:
        continue

    # Check if height is in correct range
    if 0 < height <= 8:
        break

# Print pyramid
for i in range(1, height + 1):
    # Left side
    print(' ' * (height - i), end='')
    print('#' * i, end='')

    # Middle gap
    print(' ' * 2, end='')

    # Right side
    print('#' * i)
